src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,33 @@
+package calculator
+
+class Calculator {
+
+    fun calculate(expression: String): Int {
+
+        if (expression.isBlank()) {
+            throw IllegalArgumentException("빈 값이 올 수 없습니다.")
+        }
```

Comment:
```suggestion
        require(expression.isBlank()) { "빈 값이 올 수 없습니다." }
```

kotlin에서는 `require` 함수를 통해서 `IllegalArgumentException`을 처리할 수 있습니다! ☺️

참고
- [Kotlin 예외처리](https://seosh817.tistory.com/155)